build.gradle

```diff
@@ -95,6 +95,13 @@ task integrationTest(type: Test) {
     }
 }
 
+task initialiseDatabase(type: Test) {
+    useJUnitPlatform()
+    filter {
+        includeTestsMatching '*PingerIntTest*'
+    }
+}
+
```

Comment:
just run any test which initialises the ApplicationContext and therefore runs Flyway against the docker postgres instance